tests/fixtures/Middleware/proxy-ticket-test.yml

```diff
@@ -1,27 +1,13 @@
 -
   ticket: DESKTOP_PT-21-9fp9
-  xml: |
-    <?xml version='1.0' encoding='UTF-8'?>
-    <S:Envelope xmlns:S="http://schemas.xmlsoap.org/soap/envelope/">
-        <S:Body>
-            <ns0:createRequestsResponse xmlns:ns0="http://eu.europa.ec.dgt.epoetry">
-            </ns0:createRequestsResponse>
-        </S:Body>
-    </S:Envelope>
+  xml: request-response.xml
   expectations:
     assertions:
       - strpos(request['headers'], 'ecas:ProxyTicket:\ DESKTOP_PT-21-9fp9') !== false
 
 -
   ticket: ''
-  xml: |
-    <?xml version='1.0' encoding='UTF-8'?>
-    <S:Envelope xmlns:S="http://schemas.xmlsoap.org/soap/envelope/">
-        <S:Body>
-            <ns0:createRequestsResponse xmlns:ns0="http://eu.europa.ec.dgt.epoetry">
-            </ns0:createRequestsResponse>
-        </S:Body>
-    </S:Envelope>
+  xml: request-response.xml
```

Comment:
This doesn't need to be in the fixture, it can be part of the test setup as it's basically an empty response. Let's just dump the XML in the `MiddlewareTest` class somewhere so we get rid of unnecessary files.

Reply:
Fixed.